### ISSUE TITLE
[core] Disable Gestures System fails to compile

### ISSUE THREAD

**Issue by DarkElvenAngel** [raysan5/raylib#769]
Removing this line from config.h
`#define SUPPORT_GESTURES_SYSTEM     1`
causes errors in core.c 
```
$ make all PLATFORM=PLATFORM_RPI
gcc -c core.c -O1 -Wall -std=c99 -D_DEFAULT_SOURCE -Wno-missing-braces -Werror=pointer-arith -fno-strict-aliasing -I. -Iexternal/glfw/include -Iexternal/glfw/deps/mingw -I/opt/vc/include -I/opt/vc/include/interface/vmcs_host/linux -I/opt/vc/include/interface/vcos/pthreads -I/opt/vc/include/interface/vcos/pthreads -DPLATFORM_RPI -DGRAPHICS_API_OPENGL_ES2
core.c: In function ‘EventThread’:
core.c:4516:5: error: unknown type name ‘GestureEvent’
     GestureEvent gestureEvent;
     ^~~~~~~~~~~~
core.c:4531:33: error: request for member ‘touchAction’ in something not a structure or union
                     gestureEvent.touchAction = TOUCH_MOVE;
                                 ^
core.c:4531:48: error: ‘TOUCH_MOVE’ undeclared (first use in this function)
                     gestureEvent.touchAction = TOUCH_MOVE;
                                                ^~~~~~~~~~
core.c:4531:48: note: each undeclared identifier is reported only once for each function it appears in
core.c:4539:33: error: request for member ‘touchAction’ in something not a structure or union
                     gestureEvent.touchAction = TOUCH_MOVE;
                                 ^
core.c:4556:33: error: request for member ‘touchAction’ in something not a structure or union
                     gestureEvent.touchAction = TOUCH_MOVE;
                                 ^
core.c:4563:33: error: request for member ‘touchAction’ in something not a structure or union
                     gestureEvent.touchAction = TOUCH_MOVE;
                                 ^
core.c:4602:54: error: request for member ‘touchAction’ in something not a structure or union
                     if (event.value > 0) gestureEvent.touchAction = TOUCH_DOWN;
                                                      ^
core.c:4602:69: error: ‘TOUCH_DOWN’ undeclared (first use in this function)
                     if (event.value > 0) gestureEvent.touchAction = TOUCH_DOWN;
                                                                     ^~~~~~~~~~
core.c:4603:38: error: request for member ‘touchAction’ in something not a structure or union
                     else gestureEvent.touchAction = TOUCH_UP;
                                      ^
core.c:4603:53: error: ‘TOUCH_UP’ undeclared (first use in this function)
                     else gestureEvent.touchAction = TOUCH_UP;
                                                     ^~~~~~~~
core.c:4622:29: error: request for member ‘pointCount’ in something not a structure or union
                 gestureEvent.pointCount = 0;
                             ^
core.c:4623:58: error: request for member ‘pointCount’ in something not a structure or union
                 if (touchPosition[0].x >= 0) gestureEvent.pointCount++;
                                                          ^
core.c:4624:58: error: request for member ‘pointCount’ in something not a structure or union
                 if (touchPosition[1].x >= 0) gestureEvent.pointCount++;
                                                          ^
core.c:4625:58: error: request for member ‘pointCount’ in something not a structure or union
                 if (touchPosition[2].x >= 0) gestureEvent.pointCount++;
                                                          ^
core.c:4626:58: error: request for member ‘pointCount’ in something not a structure or union
                 if (touchPosition[3].x >= 0) gestureEvent.pointCount++;
                                                          ^
core.c:4627:29: error: request for member ‘pointerId’ in something not a structure or union
                 gestureEvent.pointerId[0] = 0;
                             ^
core.c:4628:29: error: request for member ‘pointerId’ in something not a structure or union
                 gestureEvent.pointerId[1] = 1;
                             ^
core.c:4629:29: error: request for member ‘pointerId’ in something not a structure or union
                 gestureEvent.pointerId[2] = 2;
                             ^
core.c:4630:29: error: request for member ‘pointerId’ in something not a structure or union
                 gestureEvent.pointerId[3] = 3;
                             ^
core.c:4631:29: error: request for member ‘position’ in something not a structure or union
                 gestureEvent.position[0] = touchPosition[0];
                             ^
core.c:4632:29: error: request for member ‘position’ in something not a structure or union
                 gestureEvent.position[1] = touchPosition[1];
                             ^
core.c:4633:29: error: request for member ‘position’ in something not a structure or union
                 gestureEvent.position[2] = touchPosition[2];
                             ^
core.c:4634:29: error: request for member ‘position’ in something not a structure or union
                 gestureEvent.position[3] = touchPosition[3];
                             ^
Makefile:508: recipe for target 'core.o' failed
make: *** [core.o] Error 1
```

**Comment by raysan5** [comment]
Actually, this issue only happens in some platforms, not DESKTOP platform... logic should be reviewed...

**Comment by Berni8k** [comment]
Sorry this was caused by my raw event input rewrite.

Should be fixed with pull #785

**Comment by raysan5** [comment]
Verified! It works! :D
